python/Ganga/__init__.py

```diff
@@ -309,7 +309,7 @@ def getLCGRootPath():
                  filter=Ganga.Utility.Config.expandvars)
 
 lcg_config.addOption(
-    'GLITE_ENABLE', True, 'Enables/disables the support of the GLITE middleware')
+    'GLITE_ENABLE', False, 'Enables/disables the support of the GLITE middleware')
```

Comment:
Out of curiosity which groups are effected by this? LHCb and GridPP don't __need__ this feature as far as I'm aware. They can both be configured to use it but they can both be configured to run without it.
A nice message in the release notes should catch anyone else who uses this but doesn't explicitly request it.
As for ATLAS I'm assuming they're correctly sourcing the ```ATLAS.ini``` which explicitly has this enabled when I look at it.